app/views/declaration/destination_countries_supplementary.scala.html

```diff
@@ -37,8 +38,16 @@
 
         @components.page_title(Some("declaration.destinationCountries.title"))
 
+        <style>
```

Comment:
Just propose to move styles to the different file. We don't want to add those styles to every page with autocomplete.

Reply:
done